machines/attic-server/configuration.nix

```diff
@@ -0,0 +1,23 @@
+{ self, ... }:
+{ pkgs, lib, config, ... }:
+{
+
+  networking = {
+    hostName = "attic-server";
+  };
+
+  mayniklas.user.root.enable = true;
+
+  lounge-rocks = {
+    # attic.server.enable = true;
+    nix-common.enable = true;
+    cloud-provider = {
+      enable = true;
+      proxmox.enable = true;
+    };
+  };
+
+  system.stateVersion = "23.05";
```

Comment:
We might want to put the `stateVersion` into a commonly imported module

Reply:
👍 I will do that.